src/main.cpp

```diff
@@ -1043,6 +1043,9 @@ static void startGameLoop()
 				WZGameReplayOptionsHandler replayOptions;
 				NETreplaySaveStart((currentGameMode == ActivitySink::GameMode::MULTIPLAYER) ? "multiplay" : "skirmish", replayOptions, war_getMaxReplaysSaved(), (currentGameMode == ActivitySink::GameMode::MULTIPLAYER));
 			}
+			// Adjust connected timeouts for clients (which is typically larger than the value of timeout when
+			// waiting in a lobby room).
+			NETadjustConnectedTimeoutForClients();
```

Comment:
We probably want this to be at the beginning of this function, before the call to `levLoadData` (which can currently block the main thread for an indeterminate - and potentially lengthy - amount of time).

Reply:
Thanks for spotting this. Fixed.